app/assets/stylesheets/shared/datatable.scss

```diff
@@ -59,6 +59,20 @@
       .btn {
         margin-right: .25em;
       }
+
+      .btn:focus {
+        box-shadow: 0 0 0 3px $brand-focus;
```

Comment:
Can you ask UX team about this. Maybe it must be updated globally

Reply:
asked aljosha, he said its fine to leave this as it is for now